src/utils/sortHelper.js

```diff
@@ -0,0 +1,12 @@
+exports.sortByUpdatedAt = (arrayData) => {
+  const arrayDataCopy = arrayData.slice();
+
+  arrayDataCopy.sort((a, b) => {
+    const updatedAtA = new Date(a.updatedAt).getTime();
+    const updatedAtB = new Date(b.updatedAt).getTime();
+
+    return updatedAtA < updatedAtB ? -1 : updatedAtA > updatedAtB ? 1 : 0;
```

Comment:
```suggestion
    return updatedAtB - updatedAtA  ;
```

전에 멘토님께서 삼항연산자를 2개 연결시켜서 쓰는걸 지양하라고 하시더라구요
내림차순이라면 위와같이 하는건 어떨까요?

Reply:
그렇네요! 좋은 지적 감사합니다. 변경하겠습니다.